lib/base/Page.js

```diff
@@ -186,4 +186,21 @@ Page.prototype.loadPage = function(payload) {
   throw new Error('Page Records cannot be deserialized');
 };
 
+function forOwn(obj, iteratee) {
+  obj = Object(obj);
+  for (const [key, val] of Object.entries(obj)) {
+    iteratee(val, key, obj);
+  }
+}
+
+Page.prototype.toJSON = function toJSON() {
```

Comment:
Why is this needed?

Reply:
toJSON is part of api Page class. Since it was the same across all apis. I had it extracted out like `each`, `list` in Version. https://github.com/twilio/twilio-node/blob/main/lib/rest/api/v2010/account/message.js#L508-L515